libs/core/src/storage/server-error/server-error.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+
+import { DaffPersistenceService } from '../persistence.interface';
+import { DaffServerStorageServiceError } from './server-error';
+
+/**
+ * An error storage service meant to be loaded into SSR contexts.
+ * It will always throw the {@link DaffServerStorageServiceError}.
```

Comment:
Link needs update.

Reply:
not sure what you mean but I changed the name here as well